.github/workflows/pr-preview.yml

```
@@ -0,0 +1,162 @@
+          
+          # Update app name for PR
+          sed -i 's/^app = .*/app = "${{ steps.app-name.outputs.app_name }}"/' fly.toml
+          
+          # Remove consul from experimental section (if present)
+          sed -i '/enable_consul = true/d' fly.toml
+          
+          # Add auto-scaling configuration to services section
+          # Find the [[services]] section and add auto-scaling after internal_port
+          awk '
+          /^\[\[services\]\]/ { in_services = 1 }
+          /^  internal_port = / && in_services { 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)